test/cliplugin/localkms/go.mod

```diff
@@ -4,19 +4,16 @@ go 1.22.0
 
 replace github.com/sigstore/sigstore => ../../..
 
-replace github.com/sigstore/sigstore/pkg/signature/kms/cliplugin => ../../../pkg/signature/kms/cliplugin
-
 require (
 	github.com/davecgh/go-spew v1.1.1
-	github.com/sigstore/sigstore v1.8.10
-	github.com/sigstore/sigstore/pkg/signature/kms/cliplugin v0.0.0-00010101000000-000000000000
+	github.com/sigstore/sigstore v0.0.0-00010101000000-000000000000
```

Comment:
this looks strange, perhaps it doesn't matter with the `replace` on line 5?

Reply:
That's right. Pinning an actual version won't be useful, I think.